presto-parser/src/test/java/com/facebook/presto/sql/parser/TestStatementBuilder.java

```diff
@@ -233,6 +233,10 @@ public void testStatementBuilder()
         printStatement("show roles from foo");
         printStatement("show current roles");
         printStatement("show current roles from foo");
+        printStatement("show role grants");
+        printStatement("show role grants for u");
+        printStatement("show role grants for user u from foo");
+        printStatement("show role grants for role r from foo");
```

Comment:
You can't because the SqlFormatter always prints out "for". This is the same as for other SHOW commands.